package.json

```diff
@@ -17,7 +17,7 @@
     "test": "mocha -R spec -t 16000 test/*-test.js"
   },
   "dependencies": {
-    "cheerio": "~0.19.0",
+    "entities": "^1.1.1",
```

Comment:
There's currently another PR (#34) that uses html-entities instead, what do you think?